### DISCUSSION TITLE
[gatsby-plugin-image] extra div inside .gatsby-image-wrapper with unwanted padding causing image to extend whole parent container

### DISCUSSION THREAD

**Discussion by adiusz** [gatsbyjs/gatsby#31419] [Help]
Hi,
Here is my image code:
```
<StaticImage
    src="../../../static/handshake.jpg"
    layout="fullWidth"
    alt=""
    aspectRatio={3 / 1}
    formats={["auto", "webp", "avif"]}
/>
```

and there is an output in browser:
![Zrzut ekranu 2021-05-13 o 20 29 51](https://user-images.githubusercontent.com/20825919/118170068-f9a60d00-b429-11eb-8854-f665f9b65e77.png)

My question is: how can I get rid of this div (`.gatsby-image-wrapper > div[aria-hidden="true"]` selector doesn't work)? Or how can I remove this padding?

**Comment by LekoArts** [comment]
This is so that the image placeholder has the correct aspect ratio. It'll expand the image placeholder to the same size -- which is important as otherwise your image placeholder would be not visible and the proper image will "jump" in
